packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -141,7 +148,16 @@ - (void)launchURLInVC:(NSString *)urlString result:(FlutterResult)result {
   _currentSession = [[FLTUrlLaunchSession alloc] initWithUrl:url withFlutterResult:result];
   _currentSession.previousStatusBarStyle = _previousStatusBarStyle;
   safari.delegate = _currentSession;
```

Comment:
I think you can clean this code up a bit:
- Build safari inside FLTUrlLaunchSession.init instead.
- Which means, move `safari.delegate = _currentSession` to the FLTUrlLaunchSession's init.
- You can then set the safari property directly in the init.
- Then use `[_viewController presentViewController:_currentSession.safari animated......]`.